ingestion/src/kldata.rs

```diff
@@ -484,23 +474,17 @@ mod tests {
             ),
             (
                 "kldata/nationalnr=93140/type=501/unexpected",
-                Err(Error::Parse(
-                    "unexpected field in kldata header format: unexpected".to_string(),
-                )),
+                Err(ParseError::UnexpectedField("unexpected".to_string())),
                 "unexpected field",
             ),
             (
                 "kldata/messageid=10/type=501",
-                Err(Error::Parse(
-                    "missing field `nationalnr` in kldata header".to_string(),
-                )),
+                Err(ParseError::MissingField("nationalnr".to_string())),
```

Comment:
If don't have national number can we ingest as wigos or different id?

Reply:
Ah, ok. Good that we ignore, it caused issues in ODA that stuff had overlapping stationids (but from different "namespaces").